<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMultipleDownloadsDialog</class>
 <widget class="QDialog" name="AddMultipleDownloadsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add multiple downloads</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#tabWidget::pane {
border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="buttonsTabWidget">
         <attribute name="title">
          <string>Select source</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="sourceLabel">
            <property name="text">
             <string>Select the source you want to add downloads from:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="youtubePlaylistCommandLinkButton">
            <property name="text">
             <string>Youtube playlist</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="groovesharkAlbumCommandLinkButton">
            <property name="text">
             <string>Grooveshark album</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="groovesharkPlaylistCommandLinkButton">
            <property name="text">
             <string>Grooveshark playlist</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="enterSearchTermTab">
         <attribute name="title">
          <string>Enter search term</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="enterTermLabel">
            <property name="text">
             <string>Enter search term</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Widgets::ClearLineEdit" name="termLineEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="byIdCheckBox">
            <property name="text">
             <string>search by id</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="verifiedCheckBox">
            <property name="text">
             <string>retrieve only verified songs if possible</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="resultsTabWidget">
         <attribute name="title">
          <string>Results</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="styleSheet">
             <string notr="true">font-weight: bold;</string>
            </property>
            <property name="text">
             <string>Retrieving downloads ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTreeView" name="downloadsTreeView">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="moreDownloadsAvailableLabel">
            <property name="text">
             <string>there are more downloads available</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="selectAllPushButton">
        <property name="text">
         <string>Select all</string>
        </property>
        <property name="icon">
         <iconset theme="edit-select-all">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="backPushButton">
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchPushButton">
        <property name="text">
         <string>Start search</string>
        </property>
        <property name="icon">
         <iconset theme="system-search">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add selected downloads</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff/>
         </iconset>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widgets::ClearLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">martchusqtutilities/widgets/clearlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
